src/main.ts

```diff
@@ -10,6 +10,8 @@ async function bootstrap() {
 
     const app = await NestFactory.create(AppModule);
 
+    app.enableCors();
```

Comment:
jak wlaczasz corsy to moze z jakas konfiguracja?

Reply:
masz na myśli coś konkretnego? Pewnie tak, ale w sumie to nie mam nic w głowie na razie :)

Reply:
ok